### DISCUSSION TITLE
some suggestions about ”Add File Link“ function

### DISCUSSION THREAD

**Discussion by dragonlau0924** [ObsidianToAnki/Obsidian_to_Anki#554] [General]
Hello developer, thank you very much for sharing the Anki plugin. I have some suggestions. Could you please display the link passed into Anki in the "Add File Link" function as a file name instead of "Obsidian"? This would make it easier for me to know which note I have reviewed, which would be more practical.

**Comment by envico801** [comment]
Hi @dragonlau0924  , I left you a possible solution in #555
